specification/healthbot/resource-manager/Microsoft.HealthBot/stable/2023-05-01/healthbot.json

```diff
@@ -0,0 +1,950 @@
+        },
+        "x-ms-long-running-operation": true,
+        "x-ms-long-running-operation-options": {
+          "final-state-via": "azure-async-operation"
+        }
+      },
+      "get": {
+        "operationId": "Bots_Get",
+        "x-ms-examples": {
+          "ResourceInfoGet": {
+            "$ref": "./examples/ResourceInfoGet.json"
+          }
+        },
+        "description": "Get a HealthBot.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/subscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/resourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/resourceNameParameter"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "If resource is got successfully, the service should return 200.",
+            "schema": {
+              "$ref": "#/definitions/HealthBot"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed",
+            "schema": {
+              "$ref": "#/definitions/Error"
+            }
+          }
+        }
+      },
+      "patch": {
+        "operationId": "Bots_Update",
+        "x-ms-examples": {
+          "BotUpdate": {
+            "$ref": "./examples/ResourceUpdatePatch.json"
+          }
+        },
+        "description": "Patch a HealthBot.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/subscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/resourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/resourceNameParameter"
+          },
+          {
+            "name": "parameters",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/HealthBotUpdateParameters"
+            },
+            "description": "The parameters to provide for the required Azure Health Bot."
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "If resource is patched successfully, the service should return 200.",
+            "schema": {
+              "$ref": "#/definitions/HealthBot"
+            }
+          },
+          "201": {
+            "description": "If resource is received update request successfully, the service should return 201 (Created). Execution to continue asynchronously.",
+            "schema": {
+              "$ref": "#/definitions/HealthBot"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed",
+            "schema": {
+              "$ref": "#/definitions/Error"
+            }
+          }
+        },
+        "x-ms-long-running-operation": true
```

Comment:
Is this operation really become a long running operation now ? I don't see any indicators in the example response which tells customers where to get the polling url .

Reply:
@qiaozha Right, it is. We need to add something like:
```
      "headers": {
        "Azure-AsyncOperation": "https://management.azure.com/subscriptions/12345678-1234-1234-1234-123456789098/providers/Microsoft.Healthbot/locations/eastus/operationResults/742bef43-608b-4b58-ba96-6f7a32ec5f0f?api-version=2022-08-08"
    
```  }
To the 201 response, right? (we use RPaaS, and based on the doc LRO uses Azure-AsyncOperation header).